packages/maskbook/src/plugins/PluginSerivce.ts

```diff
@@ -9,3 +9,4 @@
 
 // Please make sure you have registered your plugin UI at ./PluginUI
 import './dHEDGE/messages'
+import './PoolTogether/messages'
```

Comment:
@Jack-Works Do we have any document for the new plugin infra?

Reply:
All APIs are well documented in `packages\plugin-infra\src\types.ts` and the plugin structure can be seen in `packages\plugins\example`.

Reply:
Please remove this file.

Reply:
@iRhonin hi please remove this file.

Reply:
Thanks @Jack-Works, done.